src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -452,6 +453,26 @@
     </Touch>
   </Target>
 
+  <!--
+    Copy published SBRP artifacts to reference-packages dir.
+    Artifacts get published to a temp location, as we cannot publish them directly to
+    reference-packages dir, until repo build is complete and previous content of
+    reference-packages dir is deleted.
+  -->
```

Comment:
This looks a bit hacky. We shouldn't need the previous packages anymore when we publish the new SBRPs, right? I would avoid a temporary directory and remove the previous SBRPs before publishing the new ones.

Reply:
I did consider alternatives as I don't particularly like this either. However, we cannot remove previous packages before we do inner build of SBRP repo, as those packages would be needed for the build.

Reply:
Right, we need them for the SBRP repo (inner) build but not for the repo (inner) publish. We already special case the SBRP repo in the Arcade targets that do the publishing. I would just delete the folder before we publish.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86